package.json

```diff
@@ -26,12 +26,5 @@
   },
   "devDependencies": {
     "coffeelint": "^1.10.1"
-  },
-  "activationCommands": {
-    "atom-workspace": [
-      "sync-settings:backup",
-      "sync-settings:restore",
-      "sync-settings:view-backup"
-    ]
```

Comment:
Resulting in accepted trade-off https://github.com/Hackafe/atom-sync-settings/issues/102 in favor of this PR :)